lambda_functions/execute_ssh_command_js/serverless.yml

```diff
@@ -309,44 +310,44 @@ functions:
                           --obs-year <%= obs_year %>'
     events:
       - schedule:
-          rate: cron(05 17 ? * FRI *) # Run every Friday at 3:05 AM, AEST
+          rate: cron(1 05 ? * TUE *) # Run weekly at 11:05AM AEST
```

Comment:
The `hour` and `minutes` field in the cron expression are interchanged. To schedule a lambda at 11:05AM AEST, the cron expression is:

`cron(05 01 ? * TUE *) # Run weekly at 11:05AM AEST`